app/code/Magento/WishlistAnalytics/README.md

```diff
@@ -1,3 +1,18 @@
 # Magento_WishlistAnalytics module
 
-The Magento_WishlistAnalytics module configures data definitions for a data collection related to the Wishlist module entities to be used in [Advanced Reporting](https://devdocs.magento.com/guides/v2.3/advanced-reporting/modules.html).
+The **Magento_WishlistAnalytics** module configures data definitions for a data collection related to the Wishlist module entities to be used in [Advanced Reporting](https://devdocs.magento.com/guides/v2.4/advanced-reporting/modules.html).
+
+## Installation details
+
+Before installing this module, note that the **Magento_WishlistAnalytics** is dependent on the following modules:
+
+- Magento_Analytics
+- Magento_Wishlist
+
+For information about enabling or disabling a module in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
+
+## Extensibility
```

Comment:
Where do you see it? https://github.com/magento/magento2/tree/2.4-develop/app/code/Magento/WishlistAnalytics

Reply:
@orlangur I thought that was the general topic. I removed that content. Thanks!